packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js

```diff
@@ -1,119 +1,48 @@
 import { Box, Grid } from 'grommet'
-import { observer, MobXProviderContext } from 'mobx-react'
 import { arrayOf, shape, string } from 'prop-types'
-import { useContext } from 'react'
-import styled from 'styled-components'
-import { ZooFooter } from '@zooniverse/react-components'
-import { useRouter } from 'next/router'
-import { useTranslation } from 'next-i18next'
+import { withResponsiveContext } from '@zooniverse/react-components'
 
 import { useAdminMode } from '@hooks'
-
-import { AdminContainer, Announcements, ProjectHeader } from '@components'
-import Hero from './components/Hero'
-import MessageFromResearcher from './components/MessageFromResearcher'
-import AboutProject from './components/AboutProject'
+import Hero from './components/Hero/index.js'
+import MessageFromResearcher from './components/MessageFromResearcher/index.js'
+import AboutProject from './components/AboutProject/index.js'
 import ConnectWithProject from '@shared/components/ConnectWithProject'
 import ProjectStatistics from '@shared/components/ProjectStatistics'
-import ZooniverseTalk from './components/ZooniverseTalk'
-import { Media } from '@shared/components/Media'
-import PageHeader from '../../components/PageHeader/PageHeader.js'
-
-export const adminBorderImage = 'repeating-linear-gradient(45deg,#000,#000 25px,#ff0 25px,#ff0 50px) 5'
-
-const PageBox = styled(Box)`
-  &.admin {
-    border-image: ${adminBorderImage};
-  }
-`
-
-const FullHeightBox = styled(Box)`
-  min-height: 90vh;
-`
-
-const RemainingHeightBox = styled(Box)`
-  flex-grow: 1;
-`
+import StandardLayout, { HeaderComponents } from '@shared/components/StandardLayout/StandardLayout.js'
+import ZooniverseTalk from './components/ZooniverseTalk/index.js'
 
-function useStores() {
-  const { store } = useContext(MobXProviderContext)
-  const { inBeta } = store.project
-  return {
-    inBeta
-  }
-}
-
-function ProjectHomePage({ workflows = [] }) {
-  const { inBeta } = useStores()
-  const { adminMode, toggleAdmin } = useAdminMode()
-  const router = useRouter()
-  const locale = router?.locale
-  const { t } = useTranslation()
-
-  const adminBorder = { size: 'medium' }
-  const betaBorder = { color: 'brand', size: 'medium' }
-  let border = adminMode ? adminBorder : false
-  border = inBeta ? betaBorder : border
-  const className = adminMode ? 'admin' : undefined
+function ProjectHomePage({ screenSize, workflows = [] }) {
+  const { adminMode } = useAdminMode()
+  const mobileLayout = screenSize === 'small'
```

Comment:
You can get screen size here directly from the responsive context:
https://github.com/zooniverse/front-end-monorepo/blob/48d24ab8a88e887a0a4c32c1eb0bb52b29c60b6a/packages/lib-classifier/src/components/Classifier/components/Layout/components/CenteredLayout/CenteredLayout.js#L42

Reply:
Good point! I'll update ProjectHomePage to use that method instead

Reply:
👍🏻 IIRC the `withResponsiveContext` decorator was written so that we could pass `size` down to class components.